src/current/cockroachcloud/managed-backups.md

```diff
@@ -1,23 +1,114 @@
 ---
-title: Use Managed-Service Backups
-summary: Restore data from a backup in CockroachDB Cloud.
+title: Managed Backups
+summary: Learn about CockroachDB Cloud managed backups.
 toc: true
 docs_area: manage
 ---
 
-This page describes how to use [_managed-service backups_]({% link cockroachcloud/backup-and-restore-overview.md %}) in CockroachDB {{ site.data.products.standard }}, {{ site.data.products.advanced }}, and {{ site.data.products.basic }} clusters.
+{% include cockroachcloud/backups/managed-backup-description.md %}
 
-Managed-service backups are automated backups of CockroachDB Cloud clusters that are stored in Cockroach Labs' cloud storage.
+In [{{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters](#standard-and-advanced-clusters), you can configure:
+
+- The frequency of the backups to meet [recovery point objective (RPO)]({% link {{site.current_cloud_version}}/disaster-recovery-overview.md %}#resilience-strategy) requirements.
+- The retention of the backups to set how long Cockroach Labs retains the backups.
+
+In [{{ site.data.products.basic }} clusters](#basic-clusters), you can disable managed backups.
 
 {{site.data.alerts.callout_info}}
-In addition to managed-service backups, you can configure manual backups to your own storage bucket with customer-owned backups. Refer to the [Take and Restore Customer-Owned Backups]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}) page.
+In addition to managed backups, you can take manual backups to your own storage bucket with self-managed backups. Refer to the [Take and Restore Self-Managed Backups]({% link cockroachcloud/take-and-restore-self-managed-backups.md %}) page.
 {{site.data.alerts.end}}
 
-This table outlines how frequently CockroachDB {{ site.data.products.cloud }} clusters run automated backups and the retention period for each type of backup:
+## Managed backup settings
+
+You can configure the settings of managed backups depending on the tier of your cluster.
+
+- [{{ site.data.products.basic }} clusters](#basic-clusters)
+- [{{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters](#standard-and-advanced-clusters)
+
+### Basic clusters
+
+Cockroach Labs will take a managed backup every 24 hours. You can also disable managed backups. By default, managed backups will be retained for 30 days in {{ site.data.products.basic }} clusters.
+
+Once a cluster is deleted, Cockroach Labs retains the backup for 30 days. The retained backups are not available for restore using the Cloud Console. If an organization is deleted, you will lose access to all of the managed backups that Cockroach Labs has taken of the cluster after 30 days.
```

Comment:
instead of saying its not available for restore via cloud console, could we just say "reach out to support if you want to restore a retained backup"?